.prettierignore

```diff
@@ -7,4 +7,14 @@ CHANGELOG.md
 componentUpdates.json
 componentDocs.json
 pnpm-lock.yaml
+
+# generated snippets
 **/playroom/snippets/
+
+# backwards-compatible entry points (with package.json)
+/color-mode/
+/css/
+/playroom/
+/reset/
+/test/
+/themes/
```

Comment:
Confused about this change. Wouldn't this, for example, exclude [this folder](https://github.com/seek-oss/braid-design-system/tree/001b9ae2e7ee9f8f87fd928e1f08bc755a1231a4/packages/braid-design-system/src/lib/css) from formatting?

Reply:
Because the patterns start with `/` they only apply to folders from the root.


https://user-images.githubusercontent.com/3297808/226215957-c5a875c8-353b-4605-bd4e-176ed497ed12.mp4

Reply:
Oh of course. So in this case you want to apply specifically to the root of the `braid-design-system` package?

Reply:
`.prettierignore` and `.eslintignore` are symlinked from the root of the monorep, so adding `braid-design-system` would make Prettier and ESLint work from the root of the monorepo, but not from each package. At the moment we're running Prettier, ESLint, TypeScript etc. from each package, not from the root.

Reply:
Ah, forgot about the symlinking. Makes sense now.